### PR TITLE
Rename flexvolumedriver.sh to avoid make error

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

**Comment by ldemailly** [comment]
was that it ? did the name change fix it ?
/lgtm

File: security/docker/Dockerfile.flexvolumedriver [hunk 1]
```diff
@@ -1,9 +1,9 @@
-FROM ubuntu:xenial
+FROM alpine:3.7
```
Comment (ldemailly) [inline security/docker/Dockerfile.flexvolumedriver]: why that change ?

**Comment by ldemailly** [comment]
you need to "make pull" to get the latest for circle builds to pass again

ps: why the base image change ?

File: security/docker/Dockerfile.flexvolumedriver (hunk 1 above)
Comment (wattli) [reply security/docker/Dockerfile.flexvolumedriver]: It's recommended in flexvolume.

Comment (ldemailly) [reply security/docker/Dockerfile.flexvolumedriver]: do they say why ? we build everything based on xenial so far

but actually why not make it
FROM scratch 

as if it works on alpine when built on xenial; it means it's statically linked and thus you need nothing (well you will need a statically linked bash I guess to run your script)

**Comment by ldemailly** [comment]
the only thing the script seems to be doing is copy the file around ? why do we need that inside a docker image?

File: security/docker/Dockerfile.flexvolumedriver (hunk 1 above)
Comment (wattli) [reply security/docker/Dockerfile.flexvolumedriver]: That's right, i need the bash file to run the script

Comment (ldemailly) [reply security/docker/Dockerfile.flexvolumedriver]: but the script doesn't do anything that couldn't be done outside - does it ?

**Pull request merged**